docs/lib/govuk_tech_docs/open_api/templates/responses.html.erb

```diff
@@ -53,7 +53,29 @@
           end
         end %>
 
-        <% if request_body.present? %>
+        <% example_done = false %>
+
+        <% if response.content['application/json'] %>
+          <% if response.content['application/json']["examples"] %>
+            <% response.content['application/json']["examples"].each do |name, hash| %>
```

Comment:
Would using a `response.content.dig&.each` make this easier?

Reply:
maybe i'll take a look in the next PR

Reply:
tbh i'm hoping this is throw away code. all this view code is really not ideal and should be abstracted elsewhere. i don't think this tech docs swagger combo is good and we should aim to replace it with something else. both from a technical maintenance and user perspective. I think it has great negatives for all stake holders in play.